tldr.sh

```diff
@@ -1,58 +1,56 @@
-#!/bin/bash
+#!/bin/sh
 # tldr client by Ray Lee, http://github.com/raylee/tldr
+# Improvements added by Julien Reichardt https://github.com/j8r
 # a Sunday afternoon's project, I'm sure there's room for improvement. PRs welcome!
-
-set -uf -o pipefail
```

Comment:
Because this cause some shells like `dash` (`set: Illegal option -o pipefail`) . It appears to be not POSIX compliant, and I haven't noticed problems after deleting, so I remove it.